### PR TITLE
Add edit and delete buttons in schools list

### PR DESCRIPTION
**Pull request by scara82**
This PR only add 'edit' and 'delete' button for each school that linked to the relative actions.

screen shot 2018-02-25 at 2 53 24 pm

Trello Card [https://trello.com/c/dwr2n5sR](url)

### CODE REVIEW

File: app/models/school.rb [hunk 1]
```diff
@@ -7,5 +7,5 @@ class School < ApplicationRecord
 
   validates :responsable_contact,
     presence: true,
-    length: { minimum: 9, maximum: 10 }
+    length: { minimum: 9 }
```
Comment (elle) [inline app/models/school.rb:10]: This does not belong in this PR :)

**Review by elle — APPROVED** [review]

File: app/models/school.rb (hunk 1 above)
Comment (scara82) [reply app/models/school.rb:10]: I know! 😞 By mistake I did it in the master branch and now is in all the new branches...
I had too change it because FFaker was creating phone numbers longer than 10 digits..

**Comment by scara82** [comment]
Thanks @elle  I'm merging now!
